.github/workflows/dotnet-format-md.yml

```diff
@@ -10,12 +10,13 @@ name: dotnet format
 on:
   pull_request:
     branches: [ main ]
-    paths:
-    - '**.md'
+    paths-ignore:
+    - '**.cs'
+    - '.editorconfig'
 
 jobs:
   check-format:
     runs-on: ubuntu-latest
 
     steps:
-      - run: 'echo "No build required"'
+      - run: 'echo "No build required"'
```

Comment:
At first I thought to myself... 🐇 approved a PR without a newline at the end of the file?! BUT then I opened up the resolved comment https://github.com/open-telemetry/opentelemetry-dotnet/pull/3548#discussion_r938328292 😆. #3556

Reply:
Adding a cop here #3557.